landingai/pipeline/image_source.py

```diff
@@ -122,17 +123,17 @@ class VideoFile(ImageSourceBase):
     ```
     """
 
-    def __init__(self, filename: str, samples_per_second: float = 1) -> None:
+    def __init__(self, uri: str, samples_per_second: float = 1) -> None:
```

Comment:
fyi, this is also a backward incompatible change. This one is probably fine since `VideoFile` was just recently released.
**An extended topic**: it could be beneficial to introduce the maturity level concept for SDK features/modules/APIs to our users. E.g. use alpha, beta, stable via python decorators.

Reply:
Keeping URIs as discussed